iree_tests/configs/config_sdxl_cpu_llvm_task.json

```diff
@@ -0,0 +1,17 @@
+{
+    "config_name": "sdxl_cpu_llvm_task",
+    "iree_compile_flags" : [
+      "--iree-hal-target-backends=llvm-cpu",
+      "--iree-llvmcpu-target-triple=x86_64-linux-gnu",
```

Comment:
Since this is running on the same machine that is compiling, the default of `llvm::sys::getProcessTriple()` should be fine. Prefer to not hardcode a test file to a specific platform (Linux here), since developers may want to run on other platforms (Windows in my case)